forms-flow-api/src/formsflow_api/services/application.py

```diff
@@ -360,15 +362,6 @@ def update_application(application_id: int, data: Dict, **kwargs):
             application.update(data)
         else:
             raise BusinessException("Invalid application", HTTPStatus.BAD_REQUEST)
-    
-    @staticmethod
-    def delete_application(application_id: int):
-        """Delete application."""
-        application = Application.find_by_id(application_id=application_id)
-        if application:
-            application.delete()
-        else:
-            raise BusinessException("Invalid application", HTTPStatus.BAD_REQUEST)
```

Comment:
Just moved this custom method to the end of the file for easier future upgrades.